frontend/packages/ux-editor-v3/src/components/config/Expressions/ExpressionContent/ExpressionEditMode/SimpleExpression/SubExpressionContent.tsx

```diff
@@ -60,11 +60,12 @@ export const SubExpressionContent = ({
         <Button
           title={t('right_menu.expression_sub_expression_delete')}
           color='danger'
-          icon={<TrashIcon />}
           onClick={() => onRemoveSubExpression(subExpression)}
           variant='tertiary'
           size='small'
-        />
+        >
+          <TrashIcon />
+        </Button>
```

Comment:
Denne kan vi nok bare gjøre om til en `StudioButton`. Da får vi også automatisk med stilen som gjelder for en knapp med kun ikon (siden den setter `icon={true}` på `Button`). Den ble nok glemt da vi endret alle knappene til `StudioButton`.